src/DataCollection.Shared/ViewModels/AttachmentsViewModel.cs

```diff
@@ -227,7 +227,7 @@ public ICommand OpenAttachmentCommand
                                 return;
                             }
                         }
-
+                        IsLoadingAttachments = false;
```

Comment:
Does this need to be in `try...finally` block to make sure it's set if an exception occurs?